### PR TITLE
Wordsmith obnoxiously

### PR DESCRIPTION
**Pull request by 72636c**

### CODE REVIEW

File: .changeset/rich-chairs-wink.md [hunk 1]
```diff
@@ -2,19 +2,22 @@
...
+
+The bundled patch is fairly conservative and will not attempt to migrate more complex scenarios, such as where there are other keys between the merge keys. Review whether the order of keys matters before manually updating other occurrences.
 
-This migration will not be capture all cases of this (e.g. if there are keys between the merge keys). If you have other cases, update them following the example above.
+```diff
+- - <<: *deploy
++ - <<: [*deploy, *docker]
+    label: stuff
+-   <<: *docker
+```
```
Comment (72636c) [inline .changeset/rich-chairs-wink.md:23]: I made this up. Does order matter?

File: .changeset/.PREAMBLE.md [hunk 2]
```diff
@@ -1,9 +1,9 @@
-skuba 9 is a reasonably large release, however we're hoping it won't be too much trouble to upgrade.
+skuba 9 is a reasonably large release but it shouldn't be too much trouble to upgrade.
```
Comment (72636c) [inline .changeset/.PREAMBLE.md:1]: Less hope, more SHOULD NOT

File: .changeset/dull-flowers-build.md [hunk 3]
```diff
@@ -2,16 +2,13 @@
...
-FROM --platform=arm64 node:20-alpine AS dev-deps
-
-## After
-FROM public.ecr.aws/docker/library/node:20-alpine AS dev-deps
+```diff
+- FROM --platform=arm64 node:20-alpine AS dev-deps
++ FROM public.ecr.aws/docker/library/node:20-alpine AS dev-deps
 ```
+
+Your Dockerfiles may not be set up to build multi-platform images, so keep in mind that building them locally on an Intel x86 laptop may not yield images that can execute on AWS Graviton instances.
```
Comment (72636c) [inline .changeset/dull-flowers-build.md:14]: I think this sad disclaimer is necessary? Kinda made me question if we should follow the last example here and have `_arm64` stage names instead: https://docs.docker.com/reference/build-checks/from-platform-flag-const-disallowed/

File: .changeset/big-weeks-enjoy.md [hunk 4]
```diff
@@ -2,17 +2,25 @@
...
 
-These changes may affect your project setup if customising your ESLint configuration. See the individual migration guides:
+This major upgrade bundles the following changes:
 
-- https://eslint.org/docs/latest/use/migrate-to-9.0.0
-- https://typescript-eslint.io/blog/announcing-typescript-eslint-v8
+- Migration to flat config format
 
-In addition, through these major upgrades, some lint rules have changed or have been renamed. You will likely need adjust your code after running ESLint.
+  `skuba format` will attempt to migrate your existing `.eslintignore` and `.eslintrc.js` files to a flat `eslint.config.js` file.
```
Comment (AaronMoat) [inline .changeset/big-weeks-enjoy.md:11]:
```suggestion
  `skuba format` will attempt to migrate your existing `.eslintignore` and `.eslintrc` files to a flat `eslint.config.js` file.
```

Comment (AaronMoat) [reply .changeset/big-weeks-enjoy.md:11]: wait i'm dumb nvm

File: .changeset/dull-flowers-build.md (hunk 3 above)
Comment (AaronMoat) [reply .changeset/dull-flowers-build.md:14]: 🤷

File: .changeset/.PREAMBLE.md (hunk 2 above)
Comment (samchungy) [reply .changeset/.PREAMBLE.md:1]: is this a legal disclaimer

File: .changeset/.PREAMBLE.md [hunk 5]
```diff
@@ -1,9 +1,9 @@
...
 
 The main changes are:
 
-- ESLint 9 and flat config migration (in which `skuba format` should handle most of the work for you)
+- ESLint 9 and flat config migration (where `skuba format` should handle most of the work)
 - Swapping out `ts-node` for `tsx`
-- Some fixes in Docker & Buildkite files
+- Some fixes in Buildkite & Docker files
```
Comment (samchungy) [inline .changeset/.PREAMBLE.md:7]: Sorting strikes again

File: .changeset/rich-chairs-wink.md (hunk 1 above)
Comment (AaronMoat) [reply .changeset/rich-chairs-wink.md:23]: Yes, it does; 

image

**Review by AaronMoat — APPROVED** [review]
ty! I'll release soon if no objections from @samchungy or yourself, or others :)

**Review by samchungy — APPROVED** [review]

File: .changeset/rich-chairs-wink.md (hunk 1 above)
Comment (72636c) [reply .changeset/rich-chairs-wink.md:23]: Ah, I was worried about these being different:

```yaml
config:
  - &a
    a: true
  - &b
    b: true

result:
  - <<: *a
    b: false
    <<: *b
```

```yaml
config:
  - &a
    a: true
  - &b
    b: true

result:
  - <<: [*a, *b]
    b: false
```

https://codebeautify.org/yaml-editor-online seems to indicate that the merge keys are hoisted so they are not different, if I can trust it.

File: .changeset/.PREAMBLE.md (hunk 5 above)
Comment (72636c) [reply .changeset/.PREAMBLE.md:7]: 😎

**Pull request merged**